CHANGELOG.md

```diff
@@ -2,15 +2,6 @@
 
 ## Unreleased
 
-### Build tool
-
-- Generated JavaScript functions, constants, and custom type constructors now
-  include any doc comment as a JSDoc comment, making it easier to use the
-  generated code and browse its documentation from JavaScript.
-  ([Giacomo Cavalieri](https://github.com/giacomocavalieri))
-
-## v1.11.0-rc1 - 2025-05-15
-
```

Comment:
This was duplicated and had a stray `v1.11.0-rc1` heading, my guess is it comes from a rebase gone wrong. I've deleted it!